src/structures/Player.ts

```diff
@@ -139,15 +146,37 @@ export class Player {
   ): Promise<SearchResult> {
     return this.manager.search(query, requester);
   }
+  /**
+   * Move the player to another connected node
+   * @param name
+   */
+  async movePlayer(name: string) {
+    const node = this.manager.nodes.get(name)
+    if(!node.connected) throw Error('The node is not connected');
```

Comment:
If node does not exist this will throw an error.

Reply:
i would say check if there is more than one node and check if they are connected then filter the least used nodes and connect to it any play, if there is then only you are able to change the node.